packages/domains-table/package.json

```diff
@@ -55,6 +55,7 @@
 		"webpack": "^5.89.0"
 	},
 	"peerDependencies": {
+		"@automattic/calypso-router": "workspace:^",
```

Comment:
Did you mean to put this in `dependencies` instead of `peerDependencies`?

Reply:
I intended to put it into `peerDependencies`. In these packages `page` is used only to navigate to another route, we expect someone else (the app that imports and uses the package) to set up and start the router.

Reply:
It's normal to import a peer dependency, we do it all the time. Consider React, it's the _peer_ dependency of just about every Calypso and Gutenberg package. The special thing about being a peer is that it's always the parent package's responsibility to provide the package. It's never going to be in `package/domains-table/node_modules`, it must be in one of the parent directories.